recipes/strands-agents-tools/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set python_min = "3.10" %}
+{% set name = "strands-agents-tools" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/strands_agents_tools-{{ version }}.tar.gz
+  sha256: ad8d42acbb7cf5c122da3bb09b1a745fa92b61137c0ae79a82db9e3c4ccd6e25
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling
+    - hatch-vcs
+    - pip
+    - tzdata
+  run:
+    - python >={{ python_min }}
+    - strands-agents >=0.1.0,<1.0.0
+    - rich >=14.0.0,<15.0.0
+    - sympy >=1.12.0,<2.0.0
+    - prompt-toolkit <4.0.0,>=3.0.51
+    - colorama >=0.4.6,<0.5.0
+    - aws-requests-auth >=0.4.3,<0.5.0
+    - pyjwt >=2.10.1,<3.0.0
+    - dill >=0.4.0,<0.5.0
+    - pillow >=11.2.1,<12.0.0
+    - tenacity >=9.1.2,<10.0.0
+    - watchdog >=6.0.0,<7.0.0
+    - slack-bolt >=1.23.0,<2.0.0
+    - tzdata
+
+test:
+  imports:
+    - strands_tools
+  commands:
+    - pip install tzdata --no-deps
```

Comment:
You should not be calling `pip install` here. `tzdata` should be coming from the `run` dependency.